src/content/registry/npm.js

```diff
@@ -1,12 +1,16 @@
 import { createParseCommand } from './shared';
 
-const npmInstall = /(npm|yarn)( -g)?( global)? (install|i|add|update) /;
+const npmInstall = /((npm|yarn)( -g)?( global)? (install|i|add|update))\s/;
+const npmInit = /(npx|npm init)\b/;
 const packageName = /(?<package_name>[a-z0-9_@][a-z0-9_./-]*)/;
 const packageVersion = /@(?<package_version>[~^]?\d+(\.(\d|X|x)+){0,2}(-[a-z0-9_-]+)?)/;
 const packageLabel = /@[a-z0-9_-]+/;
 const fullPackage = new RegExp(String.raw`^${packageName.source}(${packageVersion.source}|${packageLabel.source})?$`);
 
-const parsePackageString = (str) => {
+const parsePackageString = (str, baseCommand, packagesLength) => {
+  if ((baseCommand === 'npx' || baseCommand === 'npm init') && packagesLength > 0) {
```

Comment:
I think it will be better to hint why you're checking the `packagesLength`.

With `foundPackages`, I can see that you check you didn't found any package yet.

```suggestion
const parsePackageString = (str, baseCommand, foundPackages) => {
  if ((baseCommand === 'npx' || baseCommand === 'npm init') && foundPackages > 0) {
```